### DISCUSSION TITLE
Addition to unsafe-enchantments.

### DISCUSSION THREAD

**Discussion by Cypphi** [PurpurMC/Purpur#460] [__Dismissed__] [answered]
Add a **max-level** for every possible enchantment in the game so you can customize it a bit more.
(It would maybe flood to purpur.yml... _maybe make it so the first time you boot up your server with the unsafe-enchantments option it creates an **enchantments.yml** listing all enchantments?_)

**Comment by BillyGalbreath** [comment, marked as answer]
This is why I didnt want to add the feature in the first place. Its not customizable for everyone's needs. Its either on, or off. If you need to customize it further you'll have to make a plugin.

Purpur has extra anvil API: https://github.com/pl3xgaming/Purpur/blob/ver/1.17/patches/api/0019-Anvil-API.patch
